packages/core-components/src/components/AutoLogoutProvider/index.ts

```diff
@@ -0,0 +1,23 @@
+import { LAST_SEEN_ONLINE_STORAGE_KEY } from './disconnectedUsers';
+
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export * from './AutoLogoutProvider';
+export * from './ConfigBasedAutoLogoutProvider';
+
+/** @public */
+export const AUTOLOGOUT_LAST_SEEN_ONLINE_STORAGE_KEY =
```

Comment:
does it make sense to use the same name both internally and externally instead of re-exporting with a slightly different name?

Secondly is there a value of having this exported?

Reply:
I thought it was good to have it exported for other potential consumers which might want to know the last time the user visited the App. But I realize now that it might be not so easy to create awareness on the existence of this property. 
I can remove it :)